apps/core-v2-testnet/src/helpers/createTerm.ts

```diff
@@ -0,0 +1,31 @@
+import { Term } from "@elementfi/core-v2-typechain";
+import { BigNumber, Signer } from "ethers";
+import { validateAddresses } from "src/utils/validateAddresses";
+
+// Creates new term within existing term contract
+export async function createTerm(
+  signer: Signer,
+  term: Term,
+  start: BigNumber, // unix timestamp seconds
+  expiry: BigNumber, // unix timestamp seconds
```

Comment:
nit: unix timestamp seconds can just be a `number` argument. We should convert it to BigNumber when we call `lock()` instead.